app/models/activity.rb

```diff
@@ -113,6 +118,8 @@ def self.debtors
     joins(:participants).where('
       activities.is_payable
       AND
+      (activities.payment_deadline IS NULL OR activities.payment_deadline > ?)
```

Comment:
Shouldn't it be `activities.payment_deadline <= ?`? When today is later than the payment deadline, you're a debtor I would think. Now, if I set the payment deadline for an activity to somewhere in the future, it will still appear on the admin payments page, whereas setting it in the past will make it disappear.

Reply:
that is a good point. That mistake is elsewhere too.